### ISSUE TITLE
.NET Framework 4.8.1 Support

### ISSUE THREAD

**Issue by montex12** [mysteryx93/HanumanInstitute.MvvmDialogs#13]
When trying to use MvvmDialogs targeting .NET FW 4.8.1 it gives me compatibility errors.

I am using C# 11

```
error MSB4018: System.ArgumentException: Invalid framework version '4.8.1/win7-x86'.
error MSB4018:    in NuGet.Frameworks.NuGetFramework.ParseFrameworkNameParts(IFrameworkNameProvider mappings, String[] parts, String& framework, Version& version, String& profile)
```

**Comment by mysteryx93** [comment]
that's with WPF?

**Comment by montex12** [comment]
I am using many NuGets but only MvvmDialogs.Avalonia have some issues with MsBuild/NuGet

```XML
  <ItemGroup>
    <PackageReference Include="System.Net.Http" Version="4.3.4" />
    <PackageReference Include="Avalonia" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Desktop" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Diagnostics" Version="11.0.0-preview8" Condition="'$(Configuration)' == 'Debug'" />
    <PackageReference Include="Avalonia.ReactiveUI" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Browser" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Svg.Skia" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Xaml.Behaviors" Version="11.0.0-preview8" />
    <PackageReference Include="Avalonia.Controls.PanAndZoom" Version="11.0.0-preview8" />
    <PackageReference Include="LiveChartsCore.SkiaSharpView.Avalonia" Version="2.0.0-beta.700-11.0.0-preview7" />
    <PackageReference Include="Material.Icons.Avalonia" Version="2.0.0" />
    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="7.0.0" />
    <PackageReference Include="Splat.DependencyInjection.SourceGenerator" Version="1.1.93">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
    <PackageReference Include="HanumanInstitute.MvvmDialogs.Avalonia" Version="2.0.0-preview8" />
  </ItemGroup>
```

**Comment by mysteryx93** [comment]
Avalonia itself requires .NET Core 3.1

https://github.com/AvaloniaUI/Avalonia/wiki/Runtime-Requirements

**Comment by montex12** [comment]
On their website it is mentioned that Avalonia is compatible with .NET Standard 2.0.

https://docs.avaloniaui.net/

![image](https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs/assets/133593308/024a9ea5-1079-42c3-9d8c-422d9eaecd41)

From NuGet also I see it's compatible with all .NET Frameworks above 4.6.1
https://www.nuget.org/packages/Avalonia/11.0.0-preview8#supportedframeworks-body-tab

I also checked the project files in Avalonia repository and they seems to target both .NET 6.0 / .NET Standard 2.0:
`<TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>`

In my case I want to try to target Windows 7 using .NET Framework and would be nice if MvvmDialogs.Avalonia can support this as well.

**Comment by mysteryx93** [comment]
You're right.

I opened a ticket here about this discrepancy.

https://github.com/AvaloniaUI/Avalonia/issues/11538

**Comment by mysteryx93** [comment]
Fixed in v2.0-rc1
https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs/releases/tag/v2.0-rc1
